api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto

```diff
@@ -170,6 +170,19 @@ message UdpProxyConfig {
     bool propagate_response_trailers = 11;
   }
 
+  message UdpAccessLogOptions {
+    // The interval to flush access log. The UDP proxy will flush only one access log when the connection
```

Comment:
Since UDP is connection less so this comment is not accurate?

Reply:
fixed, thanks!